notebooks/ch14/bias_modeling.md

```diff
@@ -81,6 +81,43 @@ The second term in this expression, $ \text{Var}(f_\hat{\theta}(z)) $, represent
 
 The third and final term in this expression, $ \text{Var}(\epsilon) $, represents the irreducible error or the noise in the data generation and collection process. This term is small when the data generation and collection process is precise or has low variation. This term is large when the data contain large amounts of noise.
 
+We start with the mean squared error
```

Comment:
I think it would nice to include a bit more background/context for this by noting which parts are random and which parts are constant (i.e. saying that we are given a query point z and we want to find the expected square error over all possible observations of gamma and all possible training data). See the first few slides of this lecture (http://www.ds100.org/sp18/assets/lectures/lec19/19-linear_models_v2-handout.pdf) for more information.

Reply:
I think this may be addressed in the very first section under "The Bias-Variance Decomposition." I think it might be sufficient, but lmk if you think there should be more

Reply:
Okay, seems fine to me.